docs/reference/allocation.qmd

```diff
@@ -0,0 +1,45 @@
+---
+title: "Allocation"
+---
+
+# The `Allocation` struct
+Object for all information about allocation
+entry                   |  description
+-----------------       |  -----------
+
+subnetwork_ids          | The unique sorted allocation network IDs
+allocation models       | The allocation models for the main network and subnetworks corresponding to subnetwork_ids
```

Comment:
There are some 'words' in here that look like variables, like `subnetwork_ids`. I think for the documentation it is nicer if these variables are described with normal words

Reply:
I was thinking about using variable-like name so it's easier for developer to look it up.